src/main/resources/public/css/styles.css

```diff
@@ -66,33 +74,54 @@ ul.nav li a:visited,
 ul.nav li a:hover,
 ul.nav li a:active,
 ul.nav li.active a {
+	padding:0;
     background: black;
-    font-size: 20px;
+    font-size: 1em;
     font-weight: bolder;
     color: white !important;
     text-decoration: none;
     text-align: center;
 }
 
+.navbar-brand{
+	font-size: 1em;
+}
+
 #password,
 #username {
     padding-block: 10%;
     margin-block: 10%;
 }
-
-#footer-id {
+/*#footer-id {
     background: black;
     text-align: center;
     padding-top: 1%;
     padding-bottom: 1%;
     margin-block: 5%;
     border: 1px solid;
+}*/
+
+
+#footer-id {
```

Comment:
footer is unreadably small when rendered on the page.
footer is not anchored at the very bottom of the page, there is a gap about 1.5x the height of the footer.